chapters/ways_to_provide_spirv.adoc

```diff
@@ -0,0 +1,149 @@
+// Copyright 2024 The Khronos Group, Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+ifndef::chapters[:chapters:]
+ifndef::images[:images: images/]
+
+[[ways-to-provide-spirv]]
+= Ways to Provide SPIR-V
+
+This chapter is designed for anyone who wants to write a tool to inspect, consume, edit, or anything else related the the SPIR-V modules passed into Vulkan.
+
+Over the years, more and more ways have been created to pass SPIR-V down to the driver and this chapter goes over them all
+
+== vkCreateShaderModule
+
+In Vulkan 1.0 release, this was the only method there was to pass SPIR-V into Vulkan. It is a very simple workflow:
+
+[source,cpp]
+----
+VkShaderModuleCreateInfo shader_module_ci;
+shader_module_ci.pCode = spirv_source;
+shader_module_ci.codeSize = sizeof(spirv_source);
+
+VkShaderModule shader_module;
+vkCreateShaderModule(device, &shader_module_ci, NULL, &shader_module);
+
+// used to create VkPipeline
+VkPipelineShaderStageCreateInfo pipeline_stage_ci;
+pipeline_stage_ci.module = shader_module;
+----
+
+The most common issue for tools or drivers consuming there is still information missing at `vkCreateShaderModule` time
```

Comment:
Not a native speaker, but this reads odd. Should there be a ":" before the "there"?

Reply:
rephrased this, made new commit to see the changes, but thanks for helping me with my native language :laughing: